lib/updatehub/updatehub.c

```diff
@@ -315,12 +322,40 @@ static int send_request(enum coap_msgtype msgtype, enum coap_method method,
 	return ret;
 }
 
+static bool install_update_cb_sha256(void)
+{
+	u8_t image_hash[TC_SHA256_DIGEST_SIZE];
+	char buffer[3], sha256_image_dowloaded[TC_SHA256_BLOCK_SIZE + 1];
```

Comment:
We really need to fix this block size thing. This define has nothing to do with the length of the hex representation of the digest. This should be `TC_SHA256_DIGEST_SIZE*2 + 1`.